apps/patient/src/utils/general.ts

```diff
@@ -44,7 +44,7 @@ export const getFulfillmentType = (
   param?: string
 ): ExtendedFulfillmentType => {
   // We don't have COURIER fulfillment type yet, so manually check for those
-  if (pharmacyId in COMMON_COURIER_PHARMACY_IDS) {
```

Comment:
hm so this was never working?

Reply:
Yeah I think it probably wasn't. `in` does a key check